diff --git a/src/engine/shared/library/sharedFoundation/src/win32/ByteOrder.cpp b/src/engine/shared/library/sharedFoundation/src/win32/ByteOrder.cpp
index 12add8e53..989403549 100644
--- a/src/engine/shared/library/sharedFoundation/src/win32/ByteOrder.cpp
+++ b/src/engine/shared/library/sharedFoundation/src/win32/ByteOrder.cpp
@@ -10,6 +10,30 @@
 
 // ======================================================================
 
+#if defined(_M_X64)
+#include <stdlib.h>
+
+ulong ntohl(ulong netLong)
+{
+	return _byteswap_ulong(netLong);
+}
+
+ulong htonl(ulong hostLong)
+{
+	return _byteswap_ulong(hostLong);
+}
+
+ushort ntohs(ushort netShort)
+{
+	return _byteswap_ushort(netShort);
+}
+
+ushort htons(ushort hostShort)
+{
+	return _byteswap_ushort(hostShort);
+}
+#else
+
 // I'm using the arguments, but the compiler can't tell that
 #pragma warning(disable: 4100)
 
@@ -56,3 +80,5 @@ __declspec(naked) ushort htons(ushort hostShort)
 } //lint !e533 !e715 // function should return a value, argument not referenced
 
 // ======================================================================
+
+#endif
